package.json

```diff
@@ -20,9 +20,9 @@
     "node": ">=12"
   },
   "dependencies": {
+    "ibm-cloud-sdk-core": "^2.14",
     "@ibm-cloud/cloudant": "0.0.23",
-    "tough-cookie": "^4.0.0",
-    "retry-axios": "^2.4.0",
+    "retry-axios": "2.x",
```

Comment:
This will create deps disparity on `retry-axios` if core move ahead, e.g. we'll have two versions - 2.6.0 from our deps and whatever new version core will be using. With `2.x` we'll just use whatever core brings in.

Reply:
Fair enough!